pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -465,6 +468,55 @@ func init() {
 	}
 
 	err = InfraSAs.addServiceAccount(
+		InfraAttachDetachControllerServiceAccountName,
```

Comment:
this will require an update to the test fixture containing the bootstrap roles. run unit tests on this package, and it'll fail with a prompt to rerun with an envvar set to update the fixture